scripts/ci/setup-travis.sh

```diff
@@ -11,15 +11,17 @@ if [[ "$CMAKE_GENERATOR" == "" ]]; then
     export CMAKE_GENERATOR="Unix Makefiles"
 fi
 
-if [[ "${STATIC_LIBS}" == "ON" ]]; then
+if [[ "$STATIC_LIBS" == "ON" ]]; then
     export CMAKE_ARGS="$CMAKE_ARGS -DBUILD_SHARED_LIBS=OFF"
 else
     export CMAKE_ARGS="$CMAKE_ARGS -DBUILD_SHARED_LIBS=ON"
 fi
 
-if [[ "${DO_COVERAGE}" == "ON" ]]; then
+if [[ "$DO_COVERAGE" == "ON" ]]; then
     export CMAKE_ARGS="$CMAKE_ARGS -DCMAKE_C_FLAGS=\"--coverage\" -DCMAKE_CXX_FLAGS=\"--coverage\""
-    pip install --user codecov
+    curl https://bootstrap.pypa.io/get-pip.py > get-pip.py
```

Comment:
Does `python3.6-pip` not exist as a package in Ubuntu?

Reply:
No, their is python3/python3-pip which gives us Python 3.4, and python3.6, which comes from deadsnakes PPA, but no python3.6-pip =/